### DISCUSSION TITLE
Function for returning the data in  gluonts model ready format?

### DISCUSSION THREAD

**Discussion by mohammednaseefop** [awslabs/gluonts#2084] [Q&A] [answered]
Has anyone already written a function to return data in gluon-ts model-ready format?
It should accept a data frame, real, categorical columns, and frequency and return the ListDataset?
(should be able to handle multiple time series as well)

**Comment by rsnirwan** [comment, marked as answer]
`v0.10` supports `pandas.DataFrame` through `gluonts.dataset.pandas.PandasDataset`. With that you can directly start modelling without converting the data first. [Here](https://ts.gluon.ai/v0.10.x/tutorials/data_manipulation/pandasdataframes.html) is a tutorial.
